README.md

```diff
@@ -314,6 +314,36 @@ Since the [DLIO benchmark](https://github.com/argonne-lcf/dlio_benchmark) doesn
 
 *Note: Within each experiment, all training parameters such as batch size and parallelism are consistent. The team is working on publishing a detailed analysis soon.*
 
+### Checkpointing
+Checkpoint benchmarks were taken on single node setups with 48vCPU, 192 GB RAM, 512 GB SSD co-located with a storage bucket in the `us-west` region. These results were the average of 10 runs. Checkpoints were tested on models ranging in size from ~76MB to ~24GB.
```

Comment:
Does there need to be any description of what "Default" means?

Reply:
I was thinking of just naming it `No Dataflux`. Thoughts?

Reply:
Sounds good